addons/sale_project/models/project.py

```diff
@@ -338,7 +338,10 @@ def _get_sale_order_items_query(self, domain_per_model=None):
         )
 
         SaleOrderLine = self.env['sale.order.line']
-        sale_order_line_domain = [('order_id', 'any', [('analytic_account_id', 'in', self.analytic_account_id.ids)])]
+        sale_order_line_domain = [
```

Comment:
for safety, I would add a '`&`', it has happened before that code did things like `['|', ('hello', '=', 'world')] + original_domain` and that missing the '`&`' in `original_domain` would change the meaning of the original domain (but usually we should always use `expression.AND`/`expression.OR` when combining domains to prevent issue like these)

```suggestion
        sale_order_line_domain = [
            '&',
```